src/components/tabs/SettingsTab.tsx

```diff
@@ -69,90 +67,104 @@ export default function SettingsTab(props: SettingsTabProps): ReactElement {
-        }}
-      >
-        Show scale bar
-      </Checkbox>
-      <Checkbox
-        type="checkbox"
-        checked={props.showTimestamp}
-        onChange={() => {
-          props.setShowTimestamp(!props.showTimestamp);
-        }}
-      >
-        Show timestamp
-      </Checkbox>
+      <CustomCollapse label="Backdrop">
+        <SettingsContainer $indentPx={INDENT_PX}>
+          <LabeledDropdown
+            label={"Backdrop images:"}
+            selected={props.backdropKey || NO_BACKDROP.key}
+            items={backdropOptions}
+            onChange={props.setBackdropKey}
+            disabled={backdropOptions.length === 1}
+          />
+          <label>
+            <span>
+              <h3>Brightness:</h3>
+            </span>
+            <HiddenMarksSlider
+              // TODO: Add a mark at the 100% position
+              style={{ maxWidth: "200px", width: "100%" }}
+              min={50}
+              max={150}
+              step={10}
+              marks={makeAntSliderMarks([50, 100, 150])}
+              value={props.backdropBrightness}
+              onChange={props.setBackdropBrightness}
+              tooltip={{ formatter: (value) => `${value}%` }}
+            />
+          </label>
+          <label>
+            <span>
+              <h3>Saturation:</h3>
+            </span>
+            <HiddenMarksSlider
+              style={{ maxWidth: "200px", width: "100%" }}
+              min={0}
+              max={100}
+              step={10}
+              marks={makeAntSliderMarks([0, 50, 100])}
+              value={props.backdropSaturation}
+              onChange={props.setBackdropSaturation}
+              tooltip={{ formatter: (value) => `${value}%` }}
+            />
+          </label>
+        </SettingsContainer>
+      </CustomCollapse>
+      <CustomCollapse label="Objects">
+        <SettingsContainer $indentPx={INDENT_PX}>
+          <DrawModeDropdown
+            label="Filtered object color:"
+            selected={props.outOfRangeDrawSettings.mode}
+            color={props.outOfRangeDrawSettings.color}
+            onChange={(mode: DrawMode, color: Color) => {
+              props.setOutOfRangeDrawSettings({ mode, color });
+            }}
+          />
+          <DrawModeDropdown
+            label="Outlier object color:"
+            selected={props.outlierDrawSettings.mode}
+            color={props.outlierDrawSettings.color}
+            onChange={(mode: DrawMode, color: Color) => {
+              props.setOutlierDrawSettings({ mode, color });
+            }}
+          />{" "}
+          <label>
+            <span>
+              <h3>Opacity:</h3>
+            </span>
+            <HiddenMarksSlider
+              style={{ maxWidth: "200px", width: "100%" }}
+              min={0}
+              max={100}
+              value={props.objectOpacity}
+              onChange={props.setObjectOpacity}
+            />
+          </label>
+          <label>
+            <span></span>
```

Comment:
Dummy span allows checkbox to be aligned properly

Reply:
is there a way to have a SettingsContainerCheckbox that automatically does this for you?  Maybe not super necessary for this PR but if you keep making more and more checkboxes...

Reply:
it's like you almost need some kind of component wrapper that abstracts the whole "Label plus Span plus input element" thing.  probably something to think about for the future.